lib/index.js

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+/* global webpage, window, document */
+var bedrock = require('bedrock');
+var cache = require('bedrock-redis');
+var config = bedrock.config;
+var express = require('bedrock-express').express;
+var phridge = require('phridge');
+var prerender = require('prerender-node');
+var querystring = require('querystring');
+var url = require('url');
+
+require('bedrock-requirejs');
+require('bedrock-views');
+
+var api = {};
+module.exports = api;
+
+require('./config');
+
+var logger = bedrock.loggers.get('app');
+var phantom = null;
+var requestQueue = {};
+
+var args = {
+  '--load-images': false,
+  '--ignore-ssl-errors': true,
+  '--ssl-protocol': 'tlsv1.2'
+};
+
+bedrock.events.on('bedrock.start', callback => {
+  // configure whitelist / blacklist
+  if(config.prerender.whitelist) {
+    prerender.whitelisted(config.prerender.whitelist);
+  }
+  if(config.prerender.blacklist) {
+    prerender.blacklisted(config.prerender.blacklist);
+  }
+  phridge.spawn(args).then(p => {
+    phantom = p;
+    return phantom.run(function() {
+      console.log('PhantomJS Started.');
```

Comment:
Hmm, it would be nice not to run it all the time. Maybe we could add a config option to turn it on or something, but it's not a free operation... and I imagine there may be significant overhead.